docs/README.md

```diff
@@ -12,6 +12,7 @@ The posts in this folder are intended for sharing technical learnings from some
 ```bash
 ├─ architecture # Azure ML architecture information 
 ├─ media # contains images for docs
+├─ observability # contains Azure resouces observability information
```

Comment:
Should we put observability inside "architecture", together with the other architecture related docs in the root folder?

Reply:
we separated observability docs from architecture in our project as its big enough topic. But if you prefer move it to architecture, i am fine with it.